data/data/bootstrap/files/usr/local/bin/bootstrap-in-place.sh

```diff
@@ -0,0 +1,51 @@
+#!/usr/bin/env bash
+# This script is executed by bootkube.sh when installing single node with bootstrap in place
+
+if [ ! -f create-static-pods-data.done ]
+then
+    echo "Stop etcd static pod by moving the manifest"
+    mv /etc/kubernetes/manifests/etcd-member-pod.yaml /etc/kubernetes
+
+    until ! crictl ps | grep etcd
+    do
+        echo "Waiting for etcd to go down"
+        sleep 10
+    done
+
+    echo "Creating master ignition and writing it to disk"
+    # Get the master ignition from MCS
+    curl -k  -H "Accept:'application/vnd.coreos.ignition+json;version=3.1.0, */*;q=0.1'" \
+    https://localhost:22623/config/master -o /opt/openshift/master.ign
+
+    echo "Updating master ignition"
+    bootkube_podman_run \
+        --rm \
+        --privileged \
+        --volume "/var/lib/etcd:/var/lib/etcd" \
+        --volume "$PWD:/assets:z" \
+        --volume "/etc/kubernetes:/etc/kubernetes" \
+        quay.io/eranco74/cluster-bootstrap:bip \
+        bootstrap-in-place --asset-dir=/assets --ignition-path=/assets/master.ign
+    #TODO: update to the proper image once it's supported
+
+    touch create-static-pods-data.done
+fi
+
+if [ ! -f coreos-installer.done ]
+then
+    # Write image + ignition to disk
+    echo "Getting installation disk"
+    INSTALL_DISK=$(lsblk | grep disk | awk 'NR==1{print $1}')
```

Comment:
is it possible we get an empty value here? For example, if u spawn a vm with live iso without attaching another volume?

Reply:
I guess it's possible.
I'll add an ENV VAR for allowing the user to set it